newsfragments/41493.significant.rst

```diff
@@ -0,0 +1 @@
+Removed deprecated method ``task`` from ``airflow.operators.python`` operator. Please use ``python_task`` from ``airflow.decorators.python`` instead.
```

Comment:
The newsfragment will be about removing the whole class.
This specific removal you are referring to will be in provider breaking change

Reply:
I think we will have to keep it in newsfragments, so far those were "core" operators - and we have not moved them to "provider.standard" yet - and as I understand that, the standard provider will be part of the Airflow 3 migration (we will not remove old Python operators from Airflow 2 - we will add deprecation there)  so the newsfragment should be sill in Airflow - I think.